src/compiler/typescript_to_rust/azle_generate/src/main.rs

```diff
@@ -14,11 +16,31 @@ fn main() {
     let compiler_info_string = std::fs::read_to_string(compiler_info_path).unwrap();
     let compiler_info: CompilerInfo = serde_json::from_str(&compiler_info_string).unwrap();
 
+    let environment_variables = if let Some(envs) = args.get(2) {
+        envs.split(',')
+            .filter(|env_var_name| env_var_name != &"")
+            .map(|env_var_name| {
+                (
+                    env_var_name.to_string(),
+                    std::env::var(env_var_name).unwrap(),
+                )
+            })
+            .collect()
+    } else {
+        vec![]
+    };
+
     let main_js = std::fs::read_to_string("src/main.js").unwrap();
 
-    let result = generate_canister(&compiler_info.file_names, main_js, &compiler_info.plugins);
+    let lib_file = generate_canister(
+        &compiler_info.file_names,
+        main_js,
+        &compiler_info.plugins,
+        &environment_variables,
+    )
+    .to_string();
 
-    // TODO let's fix this now too
-    // TODO let's really do this
-    println!("{}", result);
+    let mut f = File::create("../src/lib.rs").expect("Unable to create file");
```

Comment:
Can we use rustfmt as a library right here? Then we could get rid of the rustfmt command line thing in the Node.js code

Reply:
Done, using a library instead